### PR TITLE
Onesignal fixes

### PR DESCRIPTION
**Pull request by CodeCombiner**

### CODE REVIEW

File: build/index.js [hunk 1]
```diff
@@ -52,3 +71,32 @@ exports.onBeforeBuild = function (devkitAPI, app, config, cb) {
...
+      gradleContents = string.replace('onesignalAppIDPlaceholder', onesignalAppID);
+      fs.writeFileAsync(filepathGradleTealeaf, gradleContents, 'utf-8')
+    })
+
+    return Promise.all([writeGradleAppFile,writeGradleTealeafFile])
+  }
+  else {
+    return Promise.resolve(true);
+  }
+}*/
```
Comment (rampr) [inline build/index.js]: this can be removed.

**Comment by rampr** [comment]
Is appId now needed in manifest.xml?
https://github.com/hashcube/onesignal/blob/gradle/android/manifest.xml#L54

**Comment by CodeCombiner** [comment]
appId is not needed, removed

**Comment by rampr** [comment]
This looks good. Will merge today

**Pull request merged**